### ISSUE TITLE
Separate models to a new repo as evercam models for multiple applications

### ISSUE THREAD

**Issue by ijunaid8989** [evercam/evercam-server#1067]

**Comment by ijunaid8989** [comment]
Follow up:

https://github.com/evercam/evercam_models

evercam models have been separated into a new repo. and working fine in any other application as a dependency..

I have removed a few methods which were out of the scope for a model. 

ConCache, HtmlSanitizer and few other operations are being removed to add them to Utils instead of models. 

We don't need to change anything, Each model will be referenced as it was being used in past. 

Also, we may need to remove/add a few other methods to make things work, which will be a part of a cleanup and testing, otherwise I have tested it in an application without ecto. It's working fine.

EvercamMedia.Repo has changed to Evercam.Repo same as for Evercam.SnapshotRepo. I deliberately didn't use the word `EvercamModel`.Repo. but we can discuss.
